projects/media-viewer/src/lib/media-viewer/toolbar/search-bar/search-bar.component.ts

```diff
@@ -63,4 +74,10 @@ export class SearchBarComponent {
         : 'Phrase not found';
     }
   }
+
+  public onInputKeyPress(e: KeyboardEvent) {
+    if (e.key === 'Escape') {
+      this.searchBarHidden.next(true);
+    }
+  }
```

Comment:
Should there be a test for this functionality? It's not in there, is it?

Reply:
I'll add one